tests/util/verify.sh

```diff
@@ -132,6 +132,7 @@ __cmp_like() {
     local expected=$2
     local ipregex="^[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+$"
     local timeregex="^([0-9]+[smhd])+$"
```

Comment:
I think `22:14:46.334153508` should match `timeregex`. The current `timeregex` is badly named ... should be changed to `durationregex`.